rcl_action/src/rcl_action/action_client.c

```diff
@@ -212,9 +214,22 @@ rcl_action_client_init(
 
   // Avoid uninitialized pointers should initialization fail
   *action_client->impl = _rcl_action_get_zero_initialized_client_impl();
+
+  // Remap/Expand the action name
+  char * resolved_action_name = NULL;
+  ret = rcl_node_resolve_name(node, action_name, allocator, false, false, &resolved_action_name);
```

Comment:
Do we need the pointer variable on stack memory?

```suggestion
  // Remap/Expand the action name
  ret = rcl_node_resolve_name(node, action_name, allocator, false, false, &action_client->impl->remapped_action_name);
```

Reply:
You are right, it is not actually needed, I removed the stack variable